README.md

```diff
@@ -38,114 +46,106 @@ The league establishes a map pool for each season. The map pool for Season 1 of
+Only players who participated in the playoffs are granted prizes. Those players can [redeem their prizes in their account page](account_prizes_path).
 
-### Reporting
+## Teams
 
-After the match finishes, either team can return to the match page and record the winner.
+### Roster Limits
 
-If the match is recorded incorrectly, please contact paul@superjova.com to get it undone.
+Teams must have at least five and at most eight rostered members in order to participate in a league.
 
-## Teams
+### Timezones
 
-### Leagues
+Teams select timezones to play in. By default, timezones for all teams are Central Time.
 
-Teams can play in multiple leagues at the same time without creating a new squad. Your team can play in both the Tuesday and Thursday night leagues if you find yourself that hungry for some Chair League.
+- **Select Eastern Time** if you want to play earlier in the evening. (EST, INDIANA)
+- **Select Central Time** if you want to play in the middle of the evening.
+- **Select Pacific Time** if you want to play later in the evening. (PST, MST, ALASKA, HAWAII, ARIZONA)
 
-### Roster Cap
+### Team Rating
 
-Teams have an 8 player roster cap. This adds a responsibility of keeping an active roster to managing your team. Roster caps also make competition easier as you now can see who you may be playing when matched each week with an opponent.
+During season play, teams acquire a rating that represents how they are doing relative to other teams, with higher rating indicating stronger performance. This rating is used to match teams against opponents of similar skill within the division. Rating increases with match wins and decreases with losses; the magnitude of change is dependent on the difference between the two teams' ratings. Rating can be viewed on a team's profile page. 
 
-### Expected Behavior
+### Battle.net
 
-Chair League is a recreational league and expects its players to behave accordingly. At the end of the day, we want to create a fun and positive environment to play in. Sportmanship is expected both ingame and out. Teams that repeatedly act disrespectful will be disbanded.
+In order to join a team, players must add their Battle.net tags to their accounts. Tags are displayed publicly on the Chair League site. [Add your Battle.net tag to your account here](https://www.chairleague.com/account/user/contact_methods).
 
-### Battle.net
+### Playing in Multiple Leagues / Teams
 
-Players must add their Battle.net tags to their accounts in order to join a team. Tags are displayed publicly on the Chair League site. [Add your Battle.net tag to your account here](https://www.chairleague.com/account/user/contact_methods).
+Teams can play in multiple leagues in a season with the same roster. Moreover, players may join and play on multiple teams, provided the teams are in different leagues and/or divisions.
```

Comment:
I dont think it is league and/or division (cant play in two divisions on one night etc)  I think it is supposed to say: " Each team may compete in one division each league, amd may play in different division in separate leagues provided the roster meets that division requirement [link]. Each individual player may participate on one team per league.